### PR TITLE
Support Rails 5.2 (plus tests)

### PR DESCRIPTION
**Pull request by smudge**
/domain @samandmoore @jmileham @effron 
/platform @samandmoore @jmileham @effron 

This vendors a 5.2-compatible version of `her` and fixes an 5.2-specific breakage in `fakeable_her`.

The files in `gemfiles/` were all regenerated by `appraisal` - looks like we bumped `nokogiri` in `Gemfile` without bumping it in the Rails version builds.

Rails 5.2 is now included in the Travis build matrix.

### CODE REVIEW

**Comment by jmileham** [comment]
wow! thanks for doing this @smudge.

<< domain platform tafn - gonna wait for build success to sign off.

**Comment by jmileham** [comment]
Cool. Can you bump the alpha version number on this branch so we can release after?

**Comment by smudge** [comment]
bumped

**Comment by jmileham** [comment]
domain lgtm platform lgtm!

File: lib/test_track_rails_client/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module TestTrackRailsClient
-  VERSION = "4.0.0.alpha12" # rubocop:disable Style/MutableConstant
+  VERSION = "4.0.0.alpha13" # rubocop:disable Style/MutableConstant
```
Comment (smudge) [inline lib/test_track_rails_client/version.rb]: A fitting version for Halloween 👻 🎃

Comment (jmileham) [reply lib/test_track_rails_client/version.rb]: It's halloween somewhere!

**Comment by jmileham** [comment]
and it's live on rubygems.

**Pull request merged**